superset-frontend/src/visualizations/presets/MainPreset.js

```diff
@@ -102,6 +107,17 @@ export default class MainPreset extends Preset {
       name: 'Legacy charts',
       presets: [new DeckGLChartPreset()],
       plugins: [
+        new AtAGlanceUserIdChartPlugin().configure({
```

Comment:
Looks like everything is under here but just wanted to double check -- is this in the right spot given that "Legacy charts" name on 107?